lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced/search-date-range-advanced.component.ts

```diff
@@ -0,0 +1,152 @@
+    @Output()
+    valid = new EventEmitter<boolean>();
+
+    form = this.formBuilder.group<SearchDateRangeAdvanced>({
+        dateRangeType: DateRangeType.ANY,
+        inLastValueType: InLastDateType.DAYS,
+        inLastValue: undefined,
+        betweenStartDate: undefined,
+        betweenEndDate: undefined
+    });
+    betweenStartDateFormControl = this.form.controls.betweenStartDate;
+    betweenEndDateFormControl = this.form.controls.betweenEndDate;
+    convertedMaxDate: Date;
+    private datePickerFormat: string;
+    private destroy$ = new Subject<void>();
+
+    readonly DateRangeType = DateRangeType;
+    readonly InLastDateType = InLastDateType;
+
+    constructor(private translate: TranslationService,
+                private formBuilder: FormBuilder,
+                @Inject(MAT_DATE_FORMATS) private dateFormatConfig) {}
+
+    ngOnInit(): void {
+        this.datePickerFormat = this.dateFormat ? this.dateFormat : DEFAULT_DATE_DISPLAY_FORMAT;
+        this.dateFormatConfig.display.dateInput = this.datePickerFormat;
+        this.convertedMaxDate = endOfDay(this.maxDate && this.maxDate !== 'today' ?
+            parse(this.maxDate, this.datePickerFormat, new Date()) : new Date());
+        this.form.controls.dateRangeType.valueChanges.pipe(takeUntil(this.destroy$))
+            .subscribe((dateRangeType) => this.updateValidators(dateRangeType));
+        this.form.valueChanges.pipe(takeUntil(this.destroy$))
+            .subscribe((searchDateRangeAdvanced) => this.onChange(searchDateRangeAdvanced));
+    }
+
+    ngOnDestroy() {
+        this.destroy$.next();
+        this.destroy$.complete();
+    }
+
+    private updateValidators(dateRangeType: DateRangeType) {
+        if (dateRangeType === DateRangeType.BETWEEN) {
+            this.betweenStartDateFormControl.setValidators(Validators.required);
+            this.betweenEndDateFormControl.setValidators(Validators.required);
+            this.form.controls.inLastValue.clearValidators();
+        } else if (dateRangeType === DateRangeType.IN_LAST) {
+            this.form.controls.inLastValue.setValidators(Validators.required);
+            this.betweenStartDateFormControl.clearValidators();
+            this.betweenEndDateFormControl.clearValidators();
+        } else if (dateRangeType === DateRangeType.ANY) {
+            this.form.controls.inLastValue.clearValidators();
+            this.betweenStartDateFormControl.clearValidators();
+            this.betweenEndDateFormControl.clearValidators();
+        }
+        this.betweenStartDateFormControl.updateValueAndValidity();
+        this.betweenEndDateFormControl.updateValueAndValidity();
+        this.form.controls.inLastValue.updateValueAndValidity();
+    }
+
+    private onChange(searchDateRangeAdvanced: Partial<SearchDateRangeAdvanced>): void {
+        let query = '';
+        if (this.form.valid) {
+            if (searchDateRangeAdvanced.dateRangeType === DateRangeType.IN_LAST) {
```

Comment:
Should happen in `SearchDateRangeAdvancedTabbedComponent`

Reply:
Updated the code accordingly. The (change) event handler in SearchDateRangeAdvancedTabbedComponent is now responsible for generating the displayLabels and queries